journal/src/main/scala/com/evolutiongaming/kafka/journal/Journals.scala

```diff
@@ -85,7 +86,56 @@ object Journals {
     }
   }
 
+  def make[
+    F[_]
+    : Async
+    : FromTry : Fail : LogOf
+    : KafkaConsumerOf : KafkaProducerOf : HeadCacheOf : RandomIdOf
+    : MeasureDuration
+    : JsonCodec
+  ](
+     config: JournalConfig,
+     origin: Option[Origin],
+     eventualJournal: EventualJournal[F],
+     journalMetrics: Option[JournalMetrics[F]],
+     conversionMetrics: Option[ConversionMetrics[F]],
+     consumerPoolSize: ConsumerPoolSize,
+     consumerPoolMetrics: Option[RecoveryConsumerPoolMetrics[F]],
+     callTimeThresholds: Journal.CallTimeThresholds
+   ): Resource[F, Journals[F]] = {
+
+    val consumer = Consumer.of[F](config.kafka.consumer, config.pollTimeout)
+
+    val headCache = {
+      if (config.headCache.enabled) {
+        HeadCacheOf[F].apply(config.kafka.consumer, eventualJournal)
+      } else {
+        Resource.pure[F, HeadCache[F]](HeadCache.empty[F])
+      }
+    }
+
+    for {
+      producer <- Producer.of[F](config.kafka.producer)
+      log <- LogOf[F].apply(Journals.getClass).toResource
+      headCache <- headCache
+      journal <- make(
+        origin,
+        producer,
+        consumer,
+        eventualJournal,
+        headCache,
+        consumerPoolSize,
+        consumerPoolMetrics,
+        log,
+        conversionMetrics
+      )
+    } yield {
+      val withLog = journal.withLog(log, callTimeThresholds)
+      journalMetrics.fold(withLog) { metrics => withLog.withMetrics(metrics) }
+    }
+  }
 
+  @deprecated("use `make`", "2023-07-26")
```

Comment:
let's be consistent with naming, let's use `apply1`

Reply:
I called it `make` because @rtar wrote somewhere in this MR that we follow this convention:

- Use `apply` if method returns MyClass.
- Use `of` if method returns F[MyClass].
- Use `make` if method returns Resource[F, MyClass].

Please come to an agreement 😄

Reply:
my point here is that we already have some naming patterns here:
* `apply: MyClass`
* `of: MyClass`

I propose to keep it…